src/routes/user/user.controller.ts

```diff
@@ -94,3 +95,19 @@ export const verifyEmailHandler = async (req: Request, res: Response, next: Next
     next(error);
   }
 };
+
+/**
+ * @param {req} req - Requests
+ * @param {res} res - Response
+ * @param {next} next - next
+ * Refresh access token controller
+ */
+export const refreshAccessTokenHandler = async (req: Request, res: Response, next: NextFunction): Promise<any> => {
+  try {
+    await verifyRefreshToken(req, res, next);
```

Comment:
we can pass the middleware directly in the route.

E.g.
`router.post('/refresh-access-token', verifyRefreshToken,  userController.refreshAccessTokenHandler);`

Reply:
Yes actually by it slipped my mind 😅.

I will update it